package.json

```diff
@@ -31,7 +31,9 @@
     "@testing-library/user-event": "^12.1.10",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-icons": "^4.2.0",
     "react-image-magnify": "^2.7.4",
+    "react-router-dom": "^5.2.0",
```

Comment:
Why is this needed? If it's for navbar routing - nama itha include panna venam. UI component mattum provide panna pothum - users avangale functionalities add pannipaanga react-router (or) whatever that suits their project setup